src/runtime/bootstrap-custom-element.ts

```diff
@@ -57,8 +60,8 @@ export const proxyCustomElement = (Cstr: any, compactMeta: d.ComponentRuntimeMet
     if (BUILD.appendChildSlotFix) {
       patchSlotAppendChild(Cstr.prototype);
     }
-    if (BUILD.scopedSlotTextContentFix) {
-      patchTextContent(Cstr.prototype, cmpMeta);
+    if (BUILD.scopedSlotTextContentFix && BUILD.scoped && cmpMeta.$flags$ & CMP_FLAGS.scopedCssEncapsulation) {
```

Comment:
This check is the same that was in the `patchTextContent()` method. Just moved it to this level so we don't have to make changes in the future when we remove these `extras` flags.